src/tests/test-data/user-sessions.json

```diff
@@ -1,8 +1,20 @@
 [
   {
-    "id" : 1,
-    "user_id" : 4,
-    "refresh_token" : "IqrTkSKmel",
-    "refresh_token_expires_at" : "2025-11-21 19:19:40.911+03"
+    "id": 7,
+    "user_id": 4,
+    "refresh_token": "IqrTkSKmel",
+    "refresh_token_expires_at": "2025-11-21 19:19:40.911+03"
+  },
+  {
+    "refresh_token": "pv-jIqfPj1",
+    "refresh_token_expires_at": "2023-11-15 20:47:30+02",
+    "user_id": 1,
+    "id": 5
+  },
+  {
+    "refresh_token": "F5tTF24K9Q",
+    "refresh_token_expires_at": "2023-11-09 11:23:54+02",
+    "user_id": 1,
+    "id": 6
```

Comment:
I think the IDs should be ordered and start with 1, for better readability.

Reply:
ID 1 has already busy